src/Runner.hs

```diff
@@ -46,10 +49,14 @@ instance Show Summary where
   show (Summary examples tried errors failures) =
     printf "Examples: %d  Tried: %d  Errors: %d  Failures: %d" examples tried errors failures
 
+-- | Sum up summaries.
+instance Semigroup Summary where
+  (Summary x1 x2 x3 x4) <> (Summary y1 y2 y3 y4) = Summary (x1 + y1) (x2 + y2) (x3 + y3) (x4 + y4)
+
 -- | Sum up summaries.
 instance Monoid Summary where
   mempty = Summary 0 0 0 0
-  (Summary x1 x2 x3 x4) `mappend` (Summary y1 y2 y3 y4) = Summary (x1 + y1) (x2 + y2) (x3 + y3) (x4 + y4)
+  mappend = (<>)
```

Comment:
We want the simplest possible code here to make it compile (it's an internal type, we don't need the `Semigroup` instance), something like:

```haskell
instance Monoid Summary where
  mempty = Summary 0 0 0 0
#if MIN_VERSION_base(4,11,0)
instance Semigroup Summary where
  (Summary x1 x2 x3 x4) <> (Summary y1 y2 y3 y4) =
#else
  (Summary x1 x2 x3 x4) `mappend` (Summary y1 y2 y3 y4) =
#endif
    Summary (x1 + y1) (x2 + y2) (x3 + y3) (x4 + y4)
```
(untested!)